__tests__/kata7.getEmployerRole.test.js

```diff
@@ -1,7 +1,11 @@
-const { getEmployerRole } = require('../src');
+const getEmployerRole = require('../src/kata7.getEmployerRole');
 
 describe('getEmployerRole', () => {
+  const employees = [{
```

Comment:
For this test, it would have been perfect to have at least another element in the array, to make sure that our function is actually choosing one employee over another.